vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/AbstractBrowserTestBase.java

```diff
@@ -530,7 +531,7 @@ private static String loadDndScript(String scriptLocation) {
             try {
                 return IOUtils.readLines(stream, StandardCharsets.UTF_8)
                         .stream().collect(Collectors.joining("\n"));
-            } catch (IOException e) {
+            } catch (UncheckedIOException e) {
```

Comment:
Since we are not handling the exception in any way, I think we can just drop the try/catch bock

Reply:
sure